spec/controllers/email_actions_controller_spec.rb

```diff
@@ -19,6 +19,24 @@
     end
   end
 
+  describe "follow_discussion" do
+    before do
+      @user = FactoryGirl.create(:user)
+      @group = FactoryGirl.create(:group)
+      @group.add_member!(@user)
+
+      @discussion = FactoryGirl.build(:discussion, group: @group)
+      DiscussionService.create(discussion: @discussion, actor: @user)
+      DiscussionReader.for(discussion: @discussion, user: @user).unfollow!
+    end
+
+    it 'follows the discussion' do
+      DiscussionReader.for(discussion: @discussion, user: @user).following?.should be false
```

Comment:
The should thing is mainly people following examples in surrounding tests i
think.
Perhaps i could try writing a script which brings all our tests up to spec
On 14/02/2015 5:05 am, "James Kiesel" <notifications@github.com> wrote:

> In spec/controllers/email_actions_controller_spec.rb
> <https://github.com/loomio/loomio/pull/1908#discussion_r24674118>:
>
> > @@ -19,6 +19,24 @@
> >      end
> >    end
> >
> > +  describe "follow_discussion" do
> > +    before do
> > +      @user = FactoryGirl.create(:user)
> > +      @group = FactoryGirl.create(:group)
> > +      @group.add_member!(@user)
> > +
> > +      @discussion = FactoryGirl.build(:discussion, group: @group)
> > +      DiscussionService.create(discussion: @discussion, actor: @user)
> > +      DiscussionReader.for(discussion: @discussion, user: @user).unfollow!
> > +    end
> > +
> > +    it 'follows the discussion' do
> > +      DiscussionReader.for(discussion: @discussion, user: @user).following?.should be false
>
> expect(DiscussionReader.for(discussion: @discussion, user: @user).following?).to be_false
>
> Hm, this .should thing's popped up frequently enough recently that I'm
> thinking we might need to put some prominent wording in the 'devving on
> Loomio' wiki around it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/1908/files#r24674118>.
>